src/main/webapp/app/exercises/programming/manage/grading/codeAnalysisMapping.ts

```diff
@@ -0,0 +1,76 @@
+import { StaticCodeAnalysisCategory, StaticCodeAnalysisCategoryState } from 'app/entities/static-code-analysis-category.model';
+
+export const Mapping: StaticCodeAnalysisCategory[] = [
```

Comment:
please mark this as a dummy object that will be replaced later by the actual default categories of the server

Reply:
using a TODO :-)